pilot/pkg/serviceregistry/kube/client.go

```diff
@@ -70,14 +70,25 @@ func ResolveConfig(kubeconfig string) (string, error) {
 
 // CreateInterface is a helper function to create Kubernetes interface
 func CreateInterface(kubeconfig string) (*rest.Config, kubernetes.Interface, error) {
-	kube, err := ResolveConfig(kubeconfig)
-	if err != nil {
-		return nil, nil, err
-	}
+	var config *rest.Config
+	var err error
 
-	config, err := clientcmd.BuildConfigFromFlags("", kube)
-	if err != nil {
-		return nil, nil, err
+	kubeconfigenv := os.Getenv("KUBECONFIG")
```

Comment:
Can you put this to a helper function name as `ResolveMultiPathConfig` or some others

Reply:
done